src/puppetlabs/puppetdb/query.clj

```diff
@@ -233,26 +233,19 @@
 (defn node-columns
   "Return the queryable set of fields and corresponding table names where they reside"
   [version]
-  (case version
```

Comment:
Oh, *right* -- I'd left that file for last (wanted to talk to Ryan about it).  I'll ping him, just ignore it for now.